radicle-cli/src/commands/issue.rs

```diff
@@ -232,13 +255,19 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
                 )?;
             }
         }
-        Operation::List => {
+        Operation::Default => panic!("default should be set earlier"),
```

Comment:
The text needs to be updated, suggestion: "default operation must be set to another"